react/modules/session.ts

```diff
@@ -1,11 +1,35 @@
-import { SessionPromise } from 'vtex.render-runtime'
+export interface KeyValue {
```

Comment:
You might not need this anymore!
Try using `vtex.session-client@1.0.0`. I know you are just updating things here, so if it doesn't work straight away, I guess it's fine to keep this file. But it's worth a try ;)

Reply:
I've just found out that this file isn't being used at all 🤯 

Actually, it is, but the only hook that's using it isn't used so 🤷